pyproject.toml

```diff
@@ -17,8 +17,8 @@ dependencies = [
     "click",
     "h5py",
     "softioc>=4.4.0",
-    "pandablocks>=0.5.3",
     "pvi>=0.6",
+    "pandablocks>=0.5.4",
```

Comment:
As there will be a (minor) API break due to the keyword arguments, shouldn't pandablocks at least be bumping its minor verson?